packages/flutter_test/lib/src/binding.dart

```diff
@@ -700,13 +727,40 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
     // The LiveTestWidgetsFlutterBinding overrides this to report the exception to the console.
   }
 
+  Future<dynamic> _handleMessage(dynamic mockMessage) async {
+    final Map<dynamic, dynamic> data =
+        (mockMessage as Map<dynamic, dynamic>)['data'] as Map<dynamic, dynamic>;
+    final String message = data['message'] as String;
+    final TextDirection textDirection = TextDirection.values[data['textDirection'] as int];
+    final dynamic assertivenessLevel = data['assertiveness'];
+    final Announcement announcement;
+    if (assertivenessLevel != null)
+    {
```

Comment:
I smell C# here :) In Dart we put the opening braces on the same line, like this:

```
if (assertivenessLevel != null) {
```

(applies throughout this PR)